src/sentry/conf/types/kafka_definition.py

```diff
@@ -57,6 +57,7 @@ class Topic(Enum):
     SNUBA_SPANS = "snuba-spans"
     BUFFERED_SEGMENTS = "buffered-segments"
     BUFFERED_SEGMENTS_DLQ = "buffered-segments-dlq"
+    TASK_WORKER = "task_worker"
```

Comment:
Added a task_worker topic here, not sure if this is the one we want, but typing requires us having a valid topic